src/Symfony/Bundle/SecurityBundle/Resources/config/security.php

```diff
@@ -100,7 +100,7 @@
             ->args([
                 service('security.token_storage'),
             ])
-            ->tag('controller.argument_value_resolver', ['priority' => 120])
+            ->tag('controller.argument_value_resolver', ['priority' => 120, 'name' => UserValueResolver::class])
```

Comment:
we could turn this one into a pinned resolver to remove it from the chain (and bump fwb to 6.3 min)

Reply:
Is https://github.com/symfony/symfony/pull/48992/commits/452320caf567ecc73a8f36942c44401ece253ada alright?

Reply:
yes!
(just please keep using replaceArgument in the pass)

Reply:
hum, tests don't agree, there might be something else to change :)

Reply:
Test controllers just type their argument `UserValueResolver` so if we pin it it is no longer in the chain and no value is provided :thinking:

Reply:
oh,so we can't do this, let's revert, sorry